### DISCUSSION TITLE
Only allow user X to add items in collection of category X

### DISCUSSION THREAD

**Discussion by divvieesterhuizen** [pocketbase/pocketbase#2124] [Q&A]
Say I create a Collection CARS, with a TITLE, CATEGORY, DESCRIPTION and MAIN IMAGE field.
I Create a Category for my Collection CARS called 'SMITH CARS INC'
Then I create a user called SMITH CARS INC
Would I be able to force that when: the user SMITH CARS INC logg's into the dashboard, that when they click on the collection CARS, and add a new record, that they can only add a new item under the category SMITH CARS INC.

**Comment by ganigeorgiev** [comment]
As mentioned earlier in the support email reply, it depends on what do you refer by "dashboard".

If you mean the Admin UI, then this is not possible because there are no admin roles at the moment. All admins act as "super-user" and have access to everything.

If you want to restrict regular user access to certain actions you can use the collections API rules. More information about the API rules you can find in https://pocketbase.io/docs/api-rules-and-filters/.

From your example, I don't understand how the category is related to the user. Is it just by the name or there is some additional relation?

Although I doubt that this is what you are looking for, here is an example API rule using just the authenticated user name and the submitted category name to perform the create check:

```js
@request.auth.name = "SMITH CARS INC" && @request.data.category.name = "SMITH CARS INC"
// or
@request.auth.name != "" && @request.auth.name = @request.data.category.name
```

**Reply by divvieesterhuizen** [reply]
So on the Admin UI, when I add a user, I add that user as SUPER ADMIN
I can't for instance add a user X than can only add items to the CARS collection.

**Reply by ganigeorgiev** [reply]
I think there is some misunderstanding or at least I'm confused.

PocketBase has 2 main account levels - admins (act as "super-user" or dev accounts) and regular app users (via [auth collections](https://pocketbase.io/docs/collections/#auth-collection)).

If you want to restrict certain actions you can do that only with your app users. If you haven't already, make sure to check:

- Auth collections docs - https://pocketbase.io/docs/collections/#auth-collection
- API rules docs - https://pocketbase.io/docs/api-rules-and-filters/
